sloam:
  cloud_topic: /forest_robot/velodyne_points
  odom_topic: /odom
  debug_mode: true
  do_destagger: false
  publish_tf: true
  two_step_optim: true
  
  # Frame IDs
  map_frame_id: map
  odom_frame_id: odom
  robot_frame_id: base_link
  
  # Detection thresholds
  beam_cluster_threshold: 0.5
  default_tree_radius: 0.2
  max_dist_to_centroid: 0.2
  
  # Distance limits
  max_ground_dist: 15.0
  min_ground_dist: 1.0
  max_lidar_dist: 20.0
  
  # Landmark sizing parameters
  min_landmark_height: 0.8
  min_landmark_size: 3
  min_vertex_size: 3
  
  # Data association
  min_tree_models: 3
  min_ground_models: 15
  rough_tree_match_thresh: 2.0
  tree_match_thresh: 0.5
  add_new_tree_thresh_dist: 1.0
  
  # Pose estimation
  num_ground_features: 5
  ground_radii_bins: 2
  ground_theta_bins: 18
  ground_match_thresh: 1.0
  ground_retain_thresh: 0.05
  features_per_tree: 10
  
  # Odometry parameters
  odom_freq_filter: 10
  min_odom_distance: 0.5
  min_sloam_z: 0.1
  
  # Segmentation parameters
  seg_lidar_fov: 22.5
  seg_lidar_w: 2048
  seg_lidar_h: 64